### DISCUSSION TITLE
Webp in Prestashop 8

### DISCUSSION THREAD

**Discussion by SA-full-stack-developer** [PrestaShop/PrestaShop#31091] [Q&A] [answered]
Hi, I updated from Prestashop 1.7 to Prestashop 8, i selected that generate all imgs in webp. I clear cache and genrate all thumbs and all imgs in webp. In product I add a webp img, but when i lokk the web all images are jpg or png, the webp images not created in my hosting. If i will go to httdocs and I search the webp images, not found anything. Sorry for my english and thank you

**Comment by Hlavtox** [comment, marked as answer]
@SA-full-stack-developer Hi, the images are WEBP, but retain JPG extension for compatibility.

I suggest waiting for Prestashop 8.1, there will be new options to generate multiple image formats properly. 👍And AVIF format as a bonus.

**Comment by SA-full-stack-developer** [comment]
Thank you very much
